wire-ios-sync-engine/Source/Synchronization/SyncAgent.swift

```diff
@@ -221,7 +224,10 @@ final class SyncAgent: NSObject, SyncAgentProtocol {
     /// Perform an incremental sync.
 
     func performIncrementalSync() async throws {
-
+        Task {
+            await conversationsMonitor.start()
```

Comment:
we start generating the conversation sync with backend tickets only after initial sync is finished, in order to avoid having conversations syncing twice (in Worker and in InitialSync)